test/unit_overapproximate.jl

```diff
@@ -153,8 +153,8 @@ for N in [Float64, Float32]
     ε = N(0.01)
     p = tovrep(overapproximate(b, ε))
     for v in vertices_list(p)
-    @test norm(v) >= N(1)
-    @test norm(v) <= N(1 + ε)
+    @test _geq(norm(v), N(1))
```

Comment:
this was needed because with the new version it was giving

```julia
  Expression: norm(v) >= N(1)
   Evaluated: 0.99999994f0 >= 1.0f0
Stacktrace:
 [1] top-level scope at /home/mforets/.julia/dev/LazySets/test/unit_overapproximate.jl:156
 [2] include(::String) at ./client.jl:439
 [3] macro expansion at /home/mforets/.julia/dev/LazySets/test/runtests.jl:140 [inlined]
 [4] macro expansion at /home/mforets/Tools/julia-1.4.1-linux-x86_64/julia-1.4.1/share/julia/stdlib/v1.4/Test/src/Test.jl:1113 [inlined]
 [5] macro expansion at /home/mforets/.julia/dev/LazySets/test/runtests.jl:140 [inlined]
 [6] macro expansion at ./util.jl:175 [inlined]
 [7] top-level scope at /home/mforets/.julia/dev/LazySets/test/runtests.jl:140
...
```

Reply:
Okay, but the second test `@test norm(v) <= N(1 + ε)` should still work, right?

Reply:
Sorry, the new test was hidden here.